CustomPlugin/CpStatusBody.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace CustomPlugin
+{
+    public class CpStatusBody
+    {
+        public string Name => System.Reflection.Assembly.GetEntryAssembly()?.GetName().Name;
```

Comment:
Is it safe to use Reflection?

Reply:
It's the best way I've found to get the application name.
If the `GetEntryAssembly()` method fails, the property accessor returns `null`.